lib/org/chromium/webidl/URLExtractor.js

```diff
@@ -9,17 +9,21 @@ foam.CLASS({
 
   properties: [
     {
+      class: 'Array',
+      of: 'RegExp',
       name: 'exclude',
       documentation: 'Excludes all URLs matching this array of RegExp.',
     },
     {
+      class: 'Array',
+      of: 'RegExp',
       name: 'include',
       documentation: 'Only include URLs matching this array of RegExp.',
     },
     {
       name: 'excludeRegexp_',
       expression: function(exclude) {
-        if (Array.isArray(exclude)) {
+        if (Array.isArray(exclude) && exclude.length > 0) {
```

Comment:
We can drop the `isArray` when it's class is `Array`, right? It should have a default factory for an array of length 0.